statemachine/aws-stepfunctions-statemachine.json

```diff
@@ -102,7 +102,7 @@
       "additionalProperties": false,
       "patternProperties": {
         ".*": {
-          "oneOf": [
+          "anyOf": [
```

Comment:
Thanks for fixing this! 
Were we getting this because "oneOf" tries to validate against all shema vs. "anyOf" checks if at least one schema is valid?

Reply:
That's right.

A schema with oneOf is valid if **exactly** one of the subschemas is valid.